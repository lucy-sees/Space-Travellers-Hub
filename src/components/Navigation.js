import '../styles/Navigation.css';
import logo from '../assets/planet.png';

const Navigation = () => (
  <nav className="navmenu-desktop">
    <img src={logo} alt="logo" className="logo-img" />
    <h1 className="logo">Space Travellers&apos; Hub</h1>
    <ul className="nav-items">
      <li className="nav-item">1</li>
      <li className="nav-item">2</li>
      <li className="nav-item">3</li>
    </ul>
  </nav>
);

export default Navigation;
