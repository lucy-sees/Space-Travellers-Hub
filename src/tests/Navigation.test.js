import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Navigation from '../components/Navigation';

test('renders Navbar correctly', () => {
  const { asFragment } = render(
    <MemoryRouter>
      <Navigation />
    </MemoryRouter>,
  );

  expect(asFragment()).toMatchSnapshot();
});
