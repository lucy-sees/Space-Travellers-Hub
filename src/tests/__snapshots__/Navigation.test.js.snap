// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navbar correctly 1`] = `
<DocumentFragment>
  <nav
    class="navmenu-desktop"
  >
    <img
      alt="logo"
      class="logo-img"
      src="planet.png"
    />
    <h1
      class="logo"
    >
      Space Travellers' Hub
    </h1>
    <ul
      class="nav-items"
    >
      <li
        class="nav-item"
      >
        <a
          aria-current="page"
          class="activeLink"
          href="/"
        />
      </li>
      <li
        class="nav-item"
      >
        <a
          class="nonActiveLink"
          href="/rockets"
        >
          Rockets
        </a>
      </li>
      <li
        class="nav-item"
      >
        <a
          class="nonActiveLink"
          href="/missions"
        >
          Missions
        </a>
      </li>
      <li
        class="nav-divider"
      >
        |
      </li>
      <li
        class="nav-item"
      >
        <a
          class="nonActiveLink"
          href="/my-profile"
        >
          My Profile
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
